.github/workflows/nightly.yml

```diff
@@ -8,6 +8,22 @@ on:
   schedule:
   - cron: '0 3 * * *'
 
+  # TODO remove
```

Comment:
?

Reply:
To force it to run now, I add this section, if it works as expected I removed before removing, otherwise we wait till 3am UTC :D